benchmark/nixlbench/infinia/README.md

```
@@ -0,0 +1,167 @@
+| Parameter | Description | Default | Example |
+|-----------|-------------|---------|---------|
+| `subtenant` | Subtenant | `"red"` | `"sub1"` |
+| `sthreads` | Number of service threads (limited by CPU cores) | `8` | `16` |
+| `num_buffers` | Pre-allocated deferred operation buffers for async ops | `512` | `1024` |
+| `num_ring_entries` | Depth of the asynchronous I/O ring buffer | `512` | `1024` |
+| `coremasks` | CPU affinity: hex ("0x0F"), list ("[0-3,8]"), or empty disables | `""` | `"0-7"` |
+| `max_retries` | BatchTask retry limit | `3` | `5` |
+
+## Example Configurations
+
+### Small Transfers
```

markdownlint-cli2 (0.22.0)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)